cnf-tests/.konflux/Dockerfile

```diff
@@ -50,7 +49,7 @@ FROM registry.redhat.io/ubi9/ubi-minimal:${RHEL_VERSION}
 RUN mkdir -p /usr/local/etc/cnf
```

Comment:
If you want to pin to a specific rhel version then you probably want to be using the rhel-els image instead of ubi, though that's not specifically related to the konflux backport

Reply:
Thanks for your comment, I think most importantly we need to follow (starting 4.19 for cnf-tests) with OCP base, we've been using minimal ubi for the previous releases so before switching to rhel-els I need to evaluate the differences and see what fits better. I agree though this is not related to konflux, thus I'll follow up if needed in a future PR.